common-test/src/main/java/org/alfresco/hxi_connector/common/test/docker/util/DockerContainers.java

```diff
@@ -107,27 +131,28 @@ public static String concatJavaOpts(String... opts)
 
     public static @NotNull String getMinimalRepoJavaOpts(PostgreSQLContainer<?> postgresContainer, GenericContainer<?> activemqContainer)
     {
-        return """
-                -Ddb.driver=org.postgresql.Driver
-                -Ddb.username=%s
-                -Ddb.password=%s
-                -Ddb.url=jdbc:postgresql://%s:5432/%s
-                -Dmessaging.broker.url="failover:(nio://%s:61616)?timeout=3000&jms.useCompression=true"
-                -Dalfresco.host=localhost
-                -Dalfresco.port=8080
-                -Dtransform.service.enabled=false
-                -Dalfresco.restApi.basicAuthScheme=true
-                -Ddeployment.method=DOCKER_COMPOSE
-                -Xms1500m -Xmx1500m
-                """.formatted(
+        return MINIMAL_REPO_JAVA_OPTS.formatted(
                 postgresContainer.getUsername(),
                 postgresContainer.getPassword(),
                 postgresContainer.getNetworkAliases().stream().findFirst().get(),
                 postgresContainer.getDatabaseName(),
-                activemqContainer.getNetworkAliases().stream().findFirst().get())
+                activemqContainer.getNetworkAliases().stream().findFirst().get(),
+                "false")
```

Comment:
could you maybe create a helper method that will take all these arguments and describe them (it is hard to understand what this "true"/"false" is)

something like

getMinimalJavaOpts(postgresPass, postgresUname, postgres..., areTransformServicesEnabled)

Reply:
but maybe thats an overkill🤔 just throwing the idea

Reply:
I was thinking about that too but am afraid it may be an overkill. 
Method names are telling quite enough for me.

Reply:
Except for `-Xms1500m -Xmx1500m` this is just a map. Maybe we could reframe it that way to also address Pawel's suggestion?
I don't think I'm likely to ever look at this code again more than a handful of times, so maybe it is overkill though :smile: